crates/store/re_chunk_store/src/dataframe.rs

```diff
@@ -196,6 +191,17 @@ pub struct ComponentColumnDescriptor {
 
     /// Whether this column represents static data.
     pub is_static: bool,
+
+    /// Whether this column represents an indicator component.
+    pub is_indicator: bool,
+
+    /// Whether this column represents a [`Clear`]-related components.
+    ///
+    /// [`Clear`]: re_types_core::archetypes::Clear
+    pub is_tombstone: bool,
+
+    /// Whether this column contains either no data or only contains null and/or empty values (`[]`).
+    pub is_semantically_empty: bool,
```

Comment:
Should we just expose `ColumnMetadata` here directly?

Reply:
Eeeh, as this is the heart of the API, I'd prefer to keep it flat, simple to inspect and simple to build by hand.